backend/handlers/deletepage.go

```
@@ -55,8 +56,29 @@ func (dp *DeletePageHandler) DeletePage(c *gin.Context) {
 		return
 	}
 
-	cmd, err := dp.db.Exec(ctx, `
-		DELETE FROM pages WHERE id = $1 AND created_by = $2
+	tx, err := dp.db.BeginTx(ctx, pgx.TxOptions{})
+	defer tx.Rollback(ctx)
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)